src/main/java/io/smartcat/cassandra/diagnostics/report/RiemannQueryReporter.java

```diff
@@ -56,36 +52,35 @@ public void report(QueryReport queryReport) {
             return;
         }
 
-        logger.debug("Sending QueryReport: execTime=" + queryReport.executionTimeInMilliseconds);
-        riemannClient().event().service(serviceName).state("ok").metric(queryReport.executionTimeInMilliseconds)
-                .attribute("client", queryReport.clientAddress).attribute("statement", queryReport.statement).ttl(10)
-                .send();
+        logger.info("Sending QueryReport: execTime{}", queryReport.executionTimeInMilliseconds);
```

Comment:
Do we want this on info level? What is a point of LogQueryReporter then? IMO leave it on debug and switch on when needed.